### ISSUE TITLE
Failed to pull image "k8s.gcr.io/kube-state-metrics/kube-state-metrics:v2.0.0"

### ISSUE THREAD

**Issue by krmao** [lensapp/lens#3728]
**Describe the bug**
Failed to pull image "k8s.gcr.io/kube-state-metrics/kube-state-metrics:v2.0.0"

**To Reproduce**
1. Right click Cluster and enable lens Metrics

**Expected behavior**
Success to pull image "k8s.gcr.io/kube-state-metrics/kube-state-metrics:v2.0.0"

**Screenshots**
![image](https://user-images.githubusercontent.com/7344437/131944143-cb363e13-7d4e-49d0-9ac0-ac2eae34f23c.png)

**Environment (please complete the following information):**
- Lens Version:
Lens: 5.1.3-latest.20210722.1
Electron: 9.4.4
Chrome: 83.0.4103.122
Node: 12.14.1
© 2021 Mirantis, Inc.

- OS: [e.g. OSX]
Macos 10.15.7

- Cluster os
Huawei cloud centos 7.9

- Installation method (e.g. snap or AppImage in Linux):

**Logs:**
```
Failed to pull image "k8s.gcr.io/kube-state-metrics/kube-state-metrics:v2.0.0": rpc error: code = Unknown desc = failed to pull and unpack image "k8s.gcr.io/kube-state-metrics/kube-state-metrics:v2.0.0": failed to resolve reference "k8s.gcr.io/kube-state-metrics/kube-state-metrics:v2.0.0": failed to do request: Head https://k8s.gcr.io/v2/kube-state-metrics/kube-state-metrics/manifests/v2.0.0: dial tcp 74.125.23.82:443: i/o timeout
```

**Kubeconfig:**
```
apiVersion: v1
clusters:
- cluster:
  name: default
contexts:
- context:
    cluster: default
    user: default
  name: default
current-context: default
kind: Config
preferences: {}
users:
- name: default
  user:
```

**Additional context**
may be China can not visit k8s.gcr.io , can change the download mirror or how can I success to use lens metrics?

![image](https://user-images.githubusercontent.com/7344437/131944498-571bddbe-8f59-457c-9e65-35bcf88c70d9.png)

**Comment by jim-docker** [comment]
Did you try editing the image name afterwards? Lens will autodetect prometheus as well.
Can you install prometheus from the helm chart?

**Comment by krmao** [comment]
I tried but no autodetect
```
    kubectl create ns lens-metrics
    
    helm install kube-state-metrics bitnami/kube-state-metrics --namespace=lens-metrics
    helm install node-exporter      bitnami/node-exporter      --namespace=lens-metrics
    helm install prometheus         bitnami/prometheus         --namespace=lens-metrics
```

**Comment by Nokel81** [comment]
The China mirror does not have macOS versions of kubectl (which will be fixed by #3956) Maybe the china firewall also blocks k8s.gcr.io too?

**Comment by krmao** [comment]
sorry I don't know, and no longer in use now.
